### PR TITLE
[DO NOT MERGE] Security vulnerability fix 

### PR DESCRIPTION
**Pull request by KarthikaPKumar**
## Contributor checklist

- [ ] Please make sure your submission complies with our [development](https://github.com/prestodb/presto/wiki/Presto-Development-Guidelines#development), [formatting](https://github.com/prestodb/presto/wiki/Presto-Development-Guidelines#formatting), [commit message](https://github.com/prestodb/presto/wiki/Review-and-Commit-guidelines#commit-formatting-and-pull-requests), and [attribution guidelines](https://github.com/prestodb/presto/wiki/Review-and-Commit-guidelines#attribution).
- [ ] PR description addresses the issue accurately and concisely.  If the change is non-trivial, a GitHub Issue is referenced.
- [ ] Documented new properties (with its default value), SQL syntax, functions, or other functionality.
- [ ] If release notes are required, they follow the [release notes guidelines](https://github.com/prestodb/presto/wiki/Release-Notes-Guidelines).
- [ ] Adequate tests were added if applicable.
- [ ] CI passed.

## Release Notes
Please follow [release notes guidelines](https://github.com/prestodb/presto/wiki/Release-Notes-Guidelines) and fill in the release notes below.

```
== RELEASE NOTES ==

General Changes
* ...
* ...

Hive Changes
* ...
* ...
```

If release note is NOT required, use:

```
== NO RELEASE NOTE ==
```

### CODE REVIEW

File: presto-spark-launcher/pom.xml [hunk 1]
```diff
@@ -38,8 +38,14 @@
 
         <dependency>
             <groupId>org.apache.commons</groupId>
-            <artifactId>commons-compress</artifactId>
-            <version>1.19</version>
+            <artifactId>commons-compress</artifactId>        
+            <version>1.26.0</version>
```
Comment (elharo) [inline presto-spark-launcher/pom.xml:42]: What's the security issue this fixes and in which version was it fixed? Is there a CVE? If it was fixed before 1.26 I'd prefer to not upgrade past 1.25 just yet, for non-security reasons.

Comment (elharo) [reply presto-spark-launcher/pom.xml:42]: FYI, the dependencies in 1.26.0 are borked. If you add this version, you also need to add explicit dependencies on commons-io and commons-codec.

**Comment by steveburnett** [comment]
* Please sign the CLA by following the link "Please click here to be authorized" in [this comment](https://github.com/prestodb/presto/pull/22003#issuecomment-1963547419). For more information, see [the CLA topic in CONTRIBUTING.md](https://github.com/prestodb/presto/blob/master/CONTRIBUTING.md#contributor-license-agreement-cla). 

* Please add a release note entry following the Presto [release notes guidelines](https://github.com/prestodb/presto/wiki/Release-Notes-Guidelines). 

  Perhaps something like: 

  ```
  == RELEASE NOTES ==
  
  Security Changes
  * Upgrade commons-compress to 1.26
  ```

  (1.25 perhaps, depending on the answer to @elharo's [comment](https://github.com/prestodb/presto/pull/22003#discussion_r1502672651).)

**Comment by steveburnett** [comment]
* Please sign the Presto CLA, and address the release note entry as mentioned in [my previous comment](https://github.com/prestodb/presto/pull/22003#issuecomment-1964436069). 

* There appears to be no change in README.md. If there is no intentional change to README.md, I suggest removing the README commits from this PR.

**Pull request closed without merging**